features/pipeline-token.feature

```diff
@@ -0,0 +1,64 @@
+@ignore
+@pipelinetoken
+Feature: Pipeline Token
+
+    A Pipeline token can be created and used in place of a password when
+    interacting with the regarding pipeline API. This would allow users to obtain valid JWTs and
+    interact with the Screwdriver API in a programmatic way.
+
+    Rules:
+        - Tokens should be specifically tied to the pipeline that issued them
+        - Tokens should have a label/name associated with them
+        - Users can add description to their issued, so they know what they're using them for
+        - User can see when a token was last used
+        - Users can revoke a token at any time
+        - No one can ever look up the raw value after initially generating the token
+
+  Background:
+    Given: "calvin" is logged in
+
+  Scenario: Generate New Pipeline Token
+    Given "calvin" created pipeline dose not own a token named "tiger"
+    When a new Pipeline token named "tiger" is generated
+    And the token is used to log in to the specific pipeline
+    Then a valid JWT is received that represents "calvin"
```

Comment:
The pipeline token does not tied to the user.
I think a valid JWT may not represent the user.

Reply:
I confirm the contents of scenario.